ipapython/certdb.py

```diff
@@ -265,7 +265,8 @@ def __init__(self, nssdir=None, dbtype='auto'):
 
         if dbtype == "dbm" and not nss_supports_dbm():
             raise ValueError(
-                "NSS is built without support of the legacy database(DBM)"
+                f"NSS is built without support of the legacy database(DBM) "
+                "directory '{nssdir}'",
```

Comment:
The second string should be f-string, otherwise no interpolation occurs.

Reply:
Hmm, I must have pushed the wrong version. I changed this from using interpolation locally. I'll push again.

Reply:
Fixed, thanks.